ui/base/src/main/res/values/styles_widgets.xml

```diff
@@ -19,6 +19,23 @@
         <item name="android:background">@drawable/btn_outline</item>
         <item name="backgroundTint">?attr/colorButtonNormal</item>
     </style>
+    <style name="Widget.GodTools.Button.Outline.Red" parent="Widget.GodTools.Button.Outline">
+        <item name="backgroundTint">@color/gt_red</item>
+        <item name="android:textColor">@color/gt_red</item>
+    </style>
+    <style name="Widget.GodTools.Button.Outline.Blue" parent="Widget.GodTools.Button.Outline">
+        <item name="backgroundTint">@color/gt_blue</item>
+    </style>
+    <style name="Widget.GodTools.Button.Borderless.Colored" parent="Widget.AppCompat.Button.Borderless">
```

Comment:
what's the difference between `Borderless` and the regular `Colored` buttons?

Reply:
@frett I created this style to match the size and structure of the Outline style.  This way they look similar to each other.